src/styles/blocks/header.scss

```diff
@@ -0,0 +1,55 @@
+@import '../utils/mixins';
+
+.header {
+ @include flex-base;
+  padding: 0 50px;
+  box-shadow: 0px 2px 4px 0px #0000000D;
+
+  &__logo {
+    padding: 10px 0;
+  }
+
+/* revome margin-bottom 3px that appears when img is inside a*/
+  &__logo-image {
+    display: block;
+  }
+}
+
+.nav {
+  
+  &__list {
+    display: flex;
+  }
+
+  &__li:not(:first-child) {
+    margin-left: 20px;
+  }
+
+  &__link {
+    display: block;
```

Comment:
**display: inline-block** would suffice! It lets you define top/bottom paddings, then why use such a radical approach display: block? :d

```suggestion
    display: inline-block;
```